advanced/am-pattern-4/values.yaml

```diff
@@ -0,0 +1,322 @@
+          # Annotations for the API Manager Gateway (WebSub) service Ingress
+          annotations:
+            kubernetes.io/ingress.class: "nginx"
+            nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+
+      # API Manager's Gateway specific configurations
+      gateway:
+        # Configure Ingress
+        ingress:
+          # Name of the IngressClass to use
+          className: ""
+          # Hostname for Gateway profile
+          hostname: "gateway.am.wso2.com"
+          # Annotations for the API Manager Gateway service Ingress
+          annotations:
+            kubernetes.io/ingress.class: "nginx"
+            nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+
+        # Number of deployment replicas
+        replicas: 2
+
+        # Kubernetes RollingUpdate strategy configurations
+        strategy:
+          rollingUpdate:
+            # The maximum number of pods that can be scheduled above the desired number of pods
+            maxSurge: 2
+            # The maximum number of pods that can be unavailable during the update
+            maxUnavailable: 0
+
+      # API Manager's Control Plane specific configurations
+      trafficmanager:
+        # Indicates whether the container is running
+        livenessProbe:
+          # Number of seconds after the container has started before liveness probes are initiated
+          initialDelaySeconds: 180
+          # How often (in seconds) to perform the probe
+          periodSeconds: 10
+        # Indicates whether the container is ready to service requests
+        readinessProbe:
+          # Number of seconds after the container has started before readiness probes are initiated
+          initialDelaySeconds: 180
+          # How often (in seconds) to perform the probe
+          periodSeconds: 10
+              # Number of deployment replicas
+        replicas: 2
+        resources:
+           # These are the minimum resource recommendations for running WSO2 API Management Control Plane deployment
+           # as per official documentation (https://apim.docs.wso2.com/en/latest/install-and-setup/install/installation-prerequisites/)
+           requests:
+             memory: "2Gi"
+             cpu: "2000m"
+           limits:
+             memory: "3Gi"
+             cpu: "3000m"
+           # JVM settings
+           # These are the resource allocation configurations associated with the JVM
+           # Refer to the official documentation for advanced details (https://apim.docs.wso2.com/en/latest/install-and-setup/install/installation-prerequisites/)
+           jvm:
+             # Resource allocation for the Java Heap
+             heap:
+                memory:
+                # Initial and minimum Heap size
+                  xms: "1024m"
+                  # Maximum Heap size
+                  xmx: "1024m"
+
+        # If the deployment configurations for the Gateway profile of WSO2 API Manager v3.2.0 (<WSO2AM>/repository/conf/deployment.toml),
```

Comment:
We are referring to APIM version `v3.2.0` here.

Reply:
yes it should be for apim 3.2.0